packages/content/tests/Unit/Content/Application/ContentDataMapper/DataMapper/ExcerptDataMapperTest.php

```diff
@@ -141,6 +141,9 @@ public function testMapWithAllExcerptData(): void
 
     public function testMapWithInvalidExcerptProperty(): void
     {
+        $this->markTestSkipped('TODO: we should implement this as soon as we implemented also unlocalized properties.');
```

Comment:
Like the test but currently skipped we should add something similar we do for templateMetadata to write the properties in localized and unlocalized correctly:

https://github.com/sulu/sulu/blob/aebcb9f253457463e96a941a0958dba43066a067/packages/content/src/Application/ContentDataMapper/DataMapper/TemplateDataMapper.php#L67-L72